brownie/runlogs/2022_10_strategist.py

```diff
@@ -194,4 +194,61 @@
 safe = ApeSafe('0xF14BBdf064E3F67f51cd9BD646aE3716aD938FDC')
 safe_tx = safe.multisend_from_receipts(txs)
 safe.sign_with_frame(safe_tx)
-r = safe.post_transaction(safe_tx)
\ No newline at end of file
+r = safe.post_transaction(safe_tx)
+
+
+# --------------------------------
+# Oct 26, 2022 Accept goverance on old contracts
+# 
+
+from world import *
+
+newgov = Contract.from_explorer("0x3cdD07c16614059e66344a7b579DAB4f9516C0b6")
+
+old_contracts = [
+  "0x9C94df9d594BA1eb94430C006c269C314B1A8281",  # CompensationClaims
+  "0xD5433168Ed0B1F7714819646606DB509D9d8EC1f",  # CompoundStrategyProxy
+  "0x9f2b18751376cF6a3432eb158Ba5F9b1AbD2F7ce",  # InitializeGovernedUpgradeabilityProxy
+  "0x3c5fe0a3922777343CBD67D3732FCdc9f2Fa6f2F",  # ThreePoolStrategyProxy
+  "0x48Cf14DeA2f5dD31c57218877195913412D3278A",  # VaultCore
+  "0x12115A32a19e4994C2BA4A5437C22CEf5ABb59C3",  # CompoundStrategyProxy
+  "0x47211B1D1F6Da45aaEE06f877266E072Cf8BaA74",  # CompoundStrategyProxy
+  "0x051CaEFA90aDf261B8E8200920C83778b7B176B6",  # InitializeGovernedUpgradeabilityProxy
```

Comment:
this one holds Aave strategy implementation

Reply:
Should be done.